security/security-core/src/main/java/io/camunda/security/configuration/SecurityConfiguration.java

```diff
@@ -48,8 +46,7 @@ public void setMultiTenancy(final MultiTenancyConfiguration multiTenancy) {
     this.multiTenancy = multiTenancy;
   }
 
-  public boolean isUnauthenticatedApiAccessAllowed() {
-    return authentication.getMethod() == AuthenticationMethod.BASIC
-        && authentication.getBasic().getAllowUnauthenticatedApiAccess();
+  public boolean isApiProtected() {
+    return !authentication.getUnprotectedApi();
```

Comment:
❓ So we can now also have unprotected api access in the case of OIDC?

Reply:
yes, once we have the oidc setup ready we can validate that too, it's less about needing this but decoupling the toggle to have an unprotected api from the auth method, as we discussed.